### DISCUSSION TITLE
Uses for SMB support?

### DISCUSSION THREAD

**Discussion by LJKraus** [X1Plus/X1Plus#334] [General]
I was interested to see that X1Plus has SMB support, but I'm wondering just how it could be used before I try to set it up.  I've had some frustrating times getting file sharing to work between Linux and my  Windows  machines, so I'd like to know what benefit there might be for file sharing with my printer.

**Comment by zippytiff** [comment]
It would be cool to save videos and timelapse to SMB...... even better would be to trick the stock UI and app to use this SMB area

This way the industrial sd card, can just focus on being the os, which means smaller cards eg 16GB vs larger

Possible ? its all linux after all ;-)
